File: app/src/main/java/com/icesoft/msdb/android/model/Series.java
package com.icesoft.msdb.android.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper=false)
public class Series extends BaseModel {
    Long id;
    String name;
    String shortname;
    Integer relevance;
    String logoUrl;
    String organizer;

}

File: app/src/main/java/com/icesoft/msdb/android/model/Series.kt
package com.icesoft.msdb.android.model

data class Series(
    val id: Long? = 0,
    val name: String? = null,
    val shortname: String? = null,
    val relevance: Int? = 0,
    val logoUrl: String? = null,
    val organizer: String? = null
): BaseModel()
